mapclassify/classifiers.py

```diff
@@ -909,7 +917,7 @@ def table(self):
         fmt = self.fmt
         return _get_table(self, fmt=fmt)
 
-    def __call__(self, *args, **kwargs):
+    def __call__(self, *args):
```

Comment:
[`find_bin()` does not accept any keywords](https://github.com/pysal/mapclassify/blob/722304258ca1976e0dfdf59d7f5a7bce4c4d293c/mapclassify/classifiers.py#L966). Would it be future-proofing to keep it as passing in a `**kwargs`? Since there are no keyword arguments (at the moment), I would think not including `**kwargs` is the cleaner solution. Do you disagree?

Reply:
It is fine 👍